cli-toolchain/usage.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Usage
+canonical: https://docs.amplify.aws/cli/usage/headless
```

Comment:
Should this be https://docs.amplify.aws/cli/start instead ?
Headless is in advanced section in the new docs.

Reply:
Good catch!